Cargo.lock

```diff
@@ -3544,12 +3544,12 @@ dependencies = [
 [[package]]
 name = "peek-poke-derive"
 version = "0.2.0"
-source = "git+https://github.com/servo/webrender#4646982f69d90f0b39b6345be4e4dbd96ce5ac05"
+source = "git+https://github.com/servo/webrender#5c3e5baf669b79a8ceff017554e7812933ca0d38"
 dependencies = [
- "proc-macro2 0.4.26",
- "quote 0.6.12",
- "syn 0.15.39",
- "synstructure 0.10.1",
+ "proc-macro2 1.0.1",
+ "quote 1.0.2",
+ "syn 1.0.3",
+ "synstructure",
  "unicode-xid 0.1.0",
```

Comment:
It would be nice to update unicode-xid in WebRender as part of this. This is the only usage of unicode-xid 0.1 left aside from the old versions of proc-macro2/syn.

Reply:
Good idea.

Reply:
I made a [PR](https://github.com/servo/webrender/pull/3776) but I had forgotten that it should be done on the mozilla-central side of things, so I'll wait for someone else to do that.

Reply:
This was done in https://github.com/servo/webrender/pull/3778.